src/components/input-field/input-field.scss

```diff
@@ -35,6 +35,29 @@ $mdc-text-field-fullwidth-bottom-line-color: $lime-text-field-bottom-line-color;
     @include mdc-states-focus-opacity(0, true);
 }
 
+label {
+    &.mdc-text-field {
+        transition: background-color 0.2s ease;
+
+        &:not(.mdc-text-field--disabled) {
+            // this has to be written this way to be able to override MD styles
+            background-color: transparent;
+
+            &:hover {
```

Comment:
This hover effect is implemented only in the limel-chip-set. Should we need to implement the hover effect in the other fields as well?

Reply:
It should be implemented in input-field which is also used in the chipset. I could see it in both places when I tested. Sounds strange that you don't see it. R U sure?